processing/src/main/java/io/druid/segment/DimensionHandlerUtils.java

```diff
@@ -206,7 +207,10 @@ private static ColumnValueSelector getColumnValueSelectorFromDimensionSpec(
     return strategyFactory.makeColumnSelectorStrategy(capabilities, selector);
   }
 
-  private static final Pattern LONG_PAT = Pattern.compile("[-|+]?\\d+");
+  public static String convertObjectToString(Object valObj)
```

Comment:
Suggested to use `java.util.Objects.toString(valObj, null)` instead. Didn't notice this method during the previous round of review